Maps.md

```diff
@@ -0,0 +1,31 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas. An Area is a 2D space with predefined  `x` and `y` boundaries and Things. Locations are entry points within an Area, where an entry point is where the player may end up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, each Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads all this data, goes through every Map's raw JSON data to create a new Map object using ObjectMakr, and finally stores it for later reference.
+
+More can be read about MapsCreatr on its [Readme](https://github.com/FullScreenShenanigans/MapsCreatr/blob/master/README.md).
+
+# AreaSpawnr
+
+AreaSpawnr is a module that works with MapsCreatr to spawn Things in a certain Area.
+
+When a new Area is entered by the player, AreaSpawnr gets that Area's information from MapsCreatr. AreaSpawnr uses this to spawn the Area's predefined Things (or PreThings) that exist in the current viewable area, or bounding box. Each PreThing has its own entry in the Area's `creation` property. PreThing attributes are defined here, such as `x` and `y` positioning. AreaSpawnr uses this information to determine which PreThings belong in the current bounding box and ultimately place them.
+
+More can be read about AreaSpawnr on its [ReadMe](https://github.com/FullScreenShenanigans/AreaSpawnr/blob/master/README.md).
+
+## Entrances and Transporters
+
+When you start in a Location, you either  start by a Thing or at raw coordinates. An entrance is a Thing where a player may end up after being transported to a Location. A transporter is a Thing that allows the player to change Areas. It links to a location, which describes where the player ends up. This description is in the Area's JSON data and includes the `x` and `y` coordinates of the location and whether the location contains an entrance Thing. So every transporter is forwardly linked to a location. When the transporter is triggered, it calls `setLocation` with that location data as the argument.
```

Comment:
> An entrance is a Thing where a player may end up

Not necessarily. All the Location cares about is that it's got a reference to the Thing. It could completely ignore the thing if its entry function doesn't care, or the Thing could be nowhere near the player.

For example, some game could define a "Chase" location where the .entrance Thing is something to be chased, and the x/y are somewhere else.